src/java.base/share/classes/java/util/BitSet.java

```diff
@@ -96,13 +96,13 @@
     /**
      * The number of words in the logical size of this BitSet.
      */
-    private transient int wordsInUse = 0;
+    private transient int wordsInUse;
 
     /**
      * Whether the size of "words" is user-specified.  If so, we assume
      * the user knows what he's doing and try harder to preserve it.
      */
-    private transient boolean sizeIsSticky = false;
```

Comment:
This change is OK, but it is not related to "8268113", do you really wants to  do these changes as part of "8268113" ?

Reply:
This is just a tiny code clean-up, so I think it's ok to go